### PR TITLE
Create $DAF-0x8Fb1a59cA2d57B51e5971A85277EFe72c4492983-bsc.png

### PR DESCRIPTION
**Pull request by daftoken**
$DAF-0x8Fb1a59cA2d57B51e5971A85277EFe72c4492983-bsc

### CODE REVIEW

**Review by zeroturningback — APPROVED** [review]
Added logo

**Comment by daftoken** [comment]
@gamalielhere

**Review by gamalielhere — CHANGES_REQUESTED** [review]
this file is invalid https://raw.githubusercontent.com/MyEtherWallet/ethereum-lists/1a4d78f6acc5093595520ba9b81e473780c54d88/master/src/icons/%24DAF-0x8Fb1a59cA2d57B51e5971A85277EFe72c4492983-bsc.png

**Comment by daftoken** [comment]
Let me know if this one works.

@gamalielhere 


> this file is invalid https://raw.githubusercontent.com/MyEtherWallet/ethereum-lists/1a4d78f6acc5093595520ba9b81e473780c54d88/master/src/icons/%24DAF-0x8Fb1a59cA2d57B51e5971A85277EFe72c4492983-bsc.png
logo



DAF_Logo

**Comment by daftoken** [comment]
logo2

**Comment by daftoken** [comment]
> this file is invalid https://raw.githubusercontent.com/MyEtherWallet/ethereum-lists/1a4d78f6acc5093595520ba9b81e473780c54d88/master/src/icons/%24DAF-0x8Fb1a59cA2d57B51e5971A85277EFe72c4492983-bsc.png

logo2

**Review by daftoken — COMMENTED** [review]
logo2

File: master/src/icons/$DAF-0x8Fb1a59cA2d57B51e5971A85277EFe72c4492983-bsc.png [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (daftoken) [inline master/src/icons/$DAF-0x8Fb1a59cA2d57B51e5971A85277EFe72c4492983-bsc.png:1]: DAF_Logo

[https://raw.githubusercontent.com/daftoken/DAF/main/logo2.png](url)

**Comment by daftoken** [comment]
@gamalielhere 

https://raw.githubusercontent.com/daftoken/DAF/main/logo2.png

**Pull request closed without merging**